extras/parse_travis_log.py

```diff
@@ -229,8 +257,8 @@ def print_static_analysis(d):
                 configure_ok, line = process_until(
                     f, 'You can now build and install NEST with')
 
-            if not make_ok and line.strip() == '+make':
-                make_ok, line = process_until(f, '+make install')
+            if line.strip() == '+make':
+                warnings, errors, line = count_warnings_errors(f)
 
             if not make_install_ok and line.startswith('+make install'):
```

Comment:
Apparently, count_warnings_errors can return the empty string in "line". Then this fails.